src/components/About/styles.scss

```
@@ -0,0 +1,125 @@
+@import "SHARED/fonts.scss";
+
+
+@media only screen and (min-width: 320px) {
+	.wrapper {
+		width: 100%;
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+		background-color: blue;
+		--portrait-spacing: 10vw;
+
+		.image {
+			width: 100%;
+			height: 38%;
+		}
+
+		.textContainer {
+			width: 100%;
+			height: 62%;
+			padding-top: var(--portrait-spacing);
+			padding-right: var(--portrait-spacing);
+			padding-bottom: 0;
+			padding-left: var(--portrait-spacing);
+			overflow-y: auto;
+			background-color: pink;
+
+			.heading {
+				margin-bottom: 1rem;
+				font-size: 1.15rem;
+			}
+
+			.subheading {
+				margin-bottom: 1.5rem;
+				font-size: calc(1.15rem * 1.62);
+				line-height: 1.5em;
+
+				&:before, &:after {
+					content: '"';
+				}
+			}
+			
+			.bodyContainer {
+				margin-bottom: var(--portrait-spacing);
+				line-height: 1.5em;
+				background-color: green;
+
+				p + p {
+					margin-top: 1em;
+				}
+			}
+		}
+	}
+}
+
+
+@media only screen and (min-width: 640px) and (max-width: 767px) {
+	.wrapper {
+		flex-direction: row;
+		--landscape_spacing: 5vw;
+
+		.image {
+			width: 50%;
+			height: 100%;
+		}
+
+		.textContainer {
+			width: 50%;
+			height: 100%;
+			padding-top: var(--landscape_spacing);
+			padding-right: var(--landscape_spacing);
+			padding-bottom: 0;
+			padding-left: var(--landscape_spacing);
+			overflow-y: auto;
+
+			.heading {
+				margin-bottom: 0.5rem;
+				font-size: calc(1rem * 0.62);
+			}
+
+			.subheading {
+				margin-bottom: 1rem;
+				font-size: 1rem;
+			}
+
+			.bodyContainer {
+				margin-bottom: 1rem;
+				font-size: 0.5rem;
+			}
+		}
+	}
+}
+
+@media only screen and (min-width: 1024px) {
+	//NO CHANGES FOR TABLET, AWAITING FEEDBACK
+}
+
+@media only screen and (min-width: 1920px) {
+	.wrapper {
+		flex-direction: row;
+
+			.image {
+				width: 50%;
+				height: 100%;
+			}
+
+			.textContainer {
+				width: 50%;
+				height: 100%;
+				padding-top: 20vh;
+				overflow-y: hidden;
+				
+				.subheading {
+					font-size: calc(1.15rem * 1.62);
+				}
+
+				.bodyContainer {
+					height: 53vh;
```

```suggestion
					height: 100%;
					max-height: 53vh;
```
We only want it to stop growing once it gets too big, we don't want to limit it more than we need to